+adi/+AD7173/Base.m

```diff
@@ -0,0 +1,155 @@
+classdef Base < adi.common.Rx & adi.common.RxTx & ...
+         matlabshared.libiio.base & adi.common.Attribute & ...
+         adi.common.RegisterReadWrite & adi.common.Channel
+    % AD7193 Precision ADC Class
+    % AD4111 is a 12 channel ADC with temperature/voltage/differential/current inputs
+
+    properties (Nontunable)
+        % SampleRate Sample Rate
+        %   Baseband sampling rate in Hz, specified as a scalar
+        %   in samples per second.
+        SampleRate = '19200'
+
+        % SamplesPerFrame Samples Per Frame
+        %   Number of samples per frame, specified as an even positive
+        %   integer.
+        SamplesPerFrame = 400
```

Comment:
Not sure I understand where the SampleRate/SamplesPerFrame default values come from?